src/app/core/side-menu/notification/invite-notif/invite-notif.component.ts

```diff
@@ -15,7 +16,8 @@ import { ErrorService } from 'app/core/error/error.service';
 })
 export class InviteNotifComponent implements OnInit {
   @Input() project: Project
-  pendingInvitations: Array<object>;
+  @Input() netConnected: boolean;
```

Comment:
just handling http errors would be enough please don't pass connection status here. if there is no connection then we should not stick at loading, some error should appear after connection timeout, 500, or any other error.